conftest.py

```diff
@@ -12,3 +16,19 @@ def pytest_configure(config):
     # XXX(dcramer): Kombu throws a warning due to transaction.commit_manually
     # being used
     warnings.filterwarnings("error", "", Warning, r"^(?!(|kombu|raven|sentry))")
+
+    # Create an empty webpack manifest file - otherwise tests will crash if it does not exist
+    os.makedirs(dist_path, exist_ok=True)
+
+    # Only create manifest if it doesn't exist
+    # (e.g. acceptance tests will have an actual manifest from webpack)
+    if os.path.exists(manifest_path):
+        return
+
+    with open(manifest_path, "w+") as fp:
+        fp.write("{}")
+
+
+def pytest_unconfigure():
+    # Clean up manifest file
+    os.remove(manifest_path)
```

Comment:
This will remove both the empty webpack manifest file as well as the real manifest generated by webpack on acceptance tests, is that ok?

Reply:
Good catch - I think that's ok - but it might just be better to address this now and backup any existing manifest files and restore them

Reply:
Updated, it'll remove only if manifest file contents are == to what we created as a placeholder.